metadata-ingestion/schedule_docs/airflow.md

```diff
@@ -0,0 +1,6 @@
+# Scheduling Ingestion using Airflow
+
+Take a look at these sample DAGs:
```

Comment:
I think we need a bit more guidance here .... Like step 1 write recipe, step 2 create a new dag step 3 give the dag a schedule, step 4 deploy dag.